SpringProject/src/main/java/com/gradproject2019/userConference/service/UserConferenceServiceImpl.java

```diff
@@ -38,11 +45,32 @@ public UserConferenceResponseDto saveInterest(UUID token, UserConferenceRequestD
         }
     }
 
+    @Override
+    public List<ConferenceResponseDto> getConferenceByUserId(UUID token, Long userId) {
+        List<ConferenceResponseDto> conferenceResponseDtos = new ArrayList<>();
+
+        checkUserAuthorised(token);
+        List<UserConference> userConferences = userConferenceRepository.findByUserId(userId);
+
+        for(UserConference userConference: userConferences) {
+            conferenceResponseDtos.add(conferenceService.getConferenceById(userConference.getConferenceId()));
```

Comment:
So this isn't a critism, working code is great at this stage, but be aware that this is inefficient code - this makes a db call for each element in this list. 

Changing this is out of scope of the project, but you'd want to select the specific columns you are returning to the FE, by joining between tables. Ask if you have any questions about this 👍